components/Search/Search.tsx

```diff
@@ -1,109 +1,148 @@
-import { useEffect, ComponentProps } from "react";
+import React, { useEffect, useRef, createElement, Fragment } from "react";
+import { autocomplete } from "@algolia/autocomplete-js";
+import "@algolia/autocomplete-theme-classic";
+import { render } from "react-dom";
 import styled from "styled-components";
-import { css, wrapper, StyledSystemWrapperProps } from "components/system";
-import Icon from "components/Icon";
+import css from "@styled-system/css";
+import Box from "components/Box";
+import { debounced } from "server/search-api-helpers";
```

Comment:
Let's not import `server/*` files on the frontend (it will include request logic and api keys to the client bundle and we don't want it). Let's move `debounced` to `utils`.